docker-hub/dvp-program.md

```diff
@@ -17,6 +17,7 @@ redirect_from:
   - /docker-hub/publish/trustchain/
   - /docker-hub/publish/byol/
   - /docker-hub/publish/publisher-center-migration/
+  - /docker-hub/publish/
 ---
```

Comment:
@aevesdocker This was because the content has actually been merged into two different files, but I will add it to here so at least it redirects somewhere and DVP is preferable.